include/aspect/boundary_composition/spherical_constant.h

```diff
@@ -87,6 +89,7 @@ namespace aspect
         /**
          * Compositions at the inner and outer boundaries.
          */
+        double surface_composition;
         double inner_composition;
```

Comment:
what does the new variable do? the class already has inner/outer boundary values, and the comment doesn't help either :-)